generate-adaptors/index.js

```diff
@@ -131,6 +131,16 @@ const sampleConfiguration = json => {
       return conf;
     });
   }
+  if (properties && required) {
+    required.forEach((key, index) => {
+      conf[key] = 'Someting';
+      conf[key] =
```

Comment:
Try
```
conf[key] = properties[key].examples?.[0]
```

What happens if there is no example and `conf[key]` is undefined?

Reply:
I think if `conf[key]` is `undefined` that credential schema is not correct and should be investigated.